cmake/InstallDirs.cmake

```diff
@@ -0,0 +1,43 @@
+# file to configure install directories
+
+# Set up install prefix
+# --------------------------------------------------------------------
+
+# make sure the install prefix has a trailing slash
+if(NOT "${CMAKE_INSTALL_PREFIX}" MATCHES "/$")
+	# shadows the cache variable with a local variable
+	set(CMAKE_INSTALL_PREFIX "${CMAKE_INSTALL_PREFIX}/")
+endif()
+
+# Set up install postfix
+# --------------------------------------------------------------------
+set(CMAKE_INSTALL_POSTFIX "" CACHE STRING "Path appended to CMAKE_INSTALL_PREFIX when ${PROJECT_NAME} is installed.  In created packages, ${PROJECT_NAME} will be installed to this path inside the package.")
+
+if((NOT ${CMAKE_INSTALL_POSTFIX} STREQUAL "") AND (NOT ${CMAKE_INSTALL_POSTFIX} STREQUAL "/"))
+	
+	# create sanitized non-cache version without a forward slash and with a trailing slash
+	if(NOT "${CMAKE_INSTALL_POSTFIX}" MATCHES "/$")
+		set(CMAKE_INSTALL_POSTFIX "${CMAKE_INSTALL_POSTFIX}/")
+	endif()
+	
+	if("${CMAKE_INSTALL_POSTFIX}" MATCHES "^/")
+		string(REGEX REPLACE "^/" "" CMAKE_INSTALL_POSTFIX "${CMAKE_INSTALL_POSTFIX}")
+	endif()
+endif()
+
+# install subdirectory setup
+# --------------------------------------------------------------------	
+
+set(BINDIR "${CMAKE_INSTALL_POSTFIX}bin") #binary subdirectory in install location
```

Comment:
A little confuse here: Why do you use `${CMAKE_INSTALL_POSTFIX}` but not `${CMAKE_INSTALL_PREFIX}`? I guess `${CMAKE_INSTALL_POSTFIX}` is not an absolute path?

Reply:
Oh, this is another Amber-specific thing that doesn't apply to QUICK; it's to do with creating certain types of .tar packages correctly.